cloud-build/cloudbuild.yaml

```diff
@@ -0,0 +1,7 @@
+steps:
+# Create docker image from regular Dockerfile
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=gcr.io/google.com/cloud-dataproc-ci/init-actions-image', '-f', 'cloud-build/Dockerfile', '.']
```

Comment:
Seem we can use variable $PROJECT_ID https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values

Reply:
$PROJECT_ID would be substituted for "google.com:cloud-dataproc-ci" but we need "google.com/cloud-dataproc-ci" as part of our GCR path. See https://cloud.google.com/container-registry/docs/overview#domain-scoped_projects.

Reply:
I have asked Andrey to create new project without `google.com:` prefix, maybe let's wait until it will be done.